docs-web/configuration/policy-resource.md

```diff
@@ -244,6 +250,52 @@ policies:
 ```
 In this example the Ingress Controller will use the configuration from the first policy reference `jwt-policy-one`, and ignores `jwt-policy-two`.
 
+### IngressMTLS
+
+The IngressMTLS policy controls client validation.
+
+For example, the following policy will verify a client using the CA certificate specified in the `ingress-mtls-secret`:
+```yaml
+ingressMTLS:
+  clientCertSecret: ingress-mtls-secret
+  verifyClient: "on"
+  verifyDepth: 1
+```
+
```

Comment:
We need to capture that the policy requires TLS termination enabled and that the policy should be referenced only in the spec.

How about adding a paragraph here:

---

A VirtualServer that references an IngressMTLS policy must:
* Enable [TLS termination](https://deploy-preview-1166--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#virtualserver-tls).
* Reference the policy in the VirtualServer [`spec`](https://deploy-preview-1166--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#virtualserver-specification). It is not allowed to reference an IngressMTLS policy in a [`route `](https://deploy-preview-1166--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#virtualserver-route) or in a VirtualServerRoute [`subroute`](https://deploy-preview-1166--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#virtualserverroute-subroute).

If the conditions above are not met, NGINX will send `500` error response to clients.